coderd/audit/audit.go

```diff
@@ -36,4 +38,6 @@ func (a *MockAuditor) Export(_ context.Context, alog database.AuditLog) error {
 	return nil
 }
 
-func (*MockAuditor) diff(any, any) Map { return Map{} }
+func (*MockAuditor) diff(any, any) Map {
+	return Map{"ttl": {"3600000000000", "7200000000000", false}}
```

Comment:
@coadler I'm attempting to mock out some diffs here so that our tests continue to pass. Is this kosher or no?

Reply:
I think it'd be confusing if all mock audit logs had a hardcoded ttl diff